src/EnergyPlus/StandardRatings.hh

```diff
@@ -450,6 +461,12 @@ namespace StandardRatings {
                                             int const PLFFPLRCurveIndex    // Index for the EIR vs part-load ratio curve
     );
 
+    Real64 CondenserEnteringFluidTemperature(
+        DataPlant::CondenserType const CondenserType,     // Chiller Condenser Type: AirCooled, WaterCooled, or EvaporativelyCooled
+        StandardRatings::AhriChillerStd const ChillerStd, // AHRI Std 550/590 (IP), or AHRI Std 551/591 (SI)
+        Real64 LoadRatio                                  // AHRI Std test load ratio: 1.0, 0.75, 0.5, 0.25
+    );
+
```

Comment:
This new function returns condenser entering fluid temperature based on the `condenser type`, `AHRI standard` and `load ratio`.